src/main/java/com/iotiq/application/service/ProductService.java

```diff
@@ -124,6 +123,8 @@ public Product getOne(UUID id) {
     @Transactional
     public ProductDto createProductForSeller(@Valid ProductCreateRequest request, Seller seller) {
         Product product = ModelMapperUtil.map(request, Product.class);
+
+        product.setBrand(request.getBrand().toUpperCase());
```

Comment:
`toUpperCase` can be used in `setBrand` method if we want the brand to always be in upper case. That way the domain logic isn't sprinkled in the code.